formats.go

```diff
@@ -109,17 +109,51 @@ func writeCertificateBundleFile(filename string, data map[string]interface{}, mo
 	}
 
 	if err := writeFile(certFile, []byte(certificate), mode); err != nil {
-		glog.Errorf("failed to write the certificate file, errro: %s", err)
+		glog.Errorf("failed to write the certificate file, error: %s", err)
+		return err
+	}
+
+	if err := writeFile(caFile, []byte(ca), mode); err != nil {
+		glog.Errorf("failed to write the ca file, error: %s", err)
+		return err
+	}
+
+	if err := writeFile(keyFile, []byte(key), mode); err != nil {
+		glog.Errorf("failed to write the key file, error: %s", err)
+		return err
+	}
+
+	return nil
+}
+
+func writeCertificateBundleChainFile(filename string, data map[string]interface{}, mode os.FileMode) error {
+	bundleChainFile := fmt.Sprintf("%s-bundle-chain.pem", filename)
```

Comment:
I think cert-chain might be the right term here, not bundle chain. Bundle implies that the private key and cert are bundled, at least in the context of vault sidekick.

Reply:
Yep I agree, I've just renamed this formatter `certchain`